setup.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python3
```

Comment:
If we want this, `/usr/bin/env python3` is more portable.

Also, the file is not executable. Need to chmod 755 and commit if this is to make sense